docs/index.md

```diff
@@ -1272,7 +1272,7 @@ $ mocha --reporter list --growl
 
 ## The `test/` Directory
 
-By default, `mocha` looks for the glob `./test/*.js` and `./test/*.coffee`, so you may want to put your tests in `test/` folder.
+By default, `mocha` looks for the glob `./test/*.js`, so you may want to put your tests in `test/` folder. If you want to include sub directories, use `--recursive`, because of `./test/*.js` is only matched with files which in the first depth of `test` directory and `./test/**/*.js` is only matched with files which in the second depth of `test` directory.
```

Comment:
> ./test/**/*.js` is only matched with files which in the second depth of `test` directory

Is this true? I though `'/**/'` matches one or more directory level.

Reply:
From https://www.npmjs.com/package/glob

> ** If a "globstar" is alone in a path portion, then it matches zero or more directories and subdirectories searching for matches. It does not crawl symlinked directories.

Reply:
This depends on if you pass a glob pattern as a string or use shell expansion. Shell expansion will only expand to the second level, and actually **exactly** the second level, not the first one, so that is also a bit confusing to many